lesson11/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <div id="js-loading"></div>
+      <ul id="js-ul"></ul>
+    </div>
+    <script type="text/javascript" src="./main.js"></script>
```

Comment:
IMO
I was taught that it is better to load the script in the head with the defer attribute for better performance,
since html parsing is not interrupted by the loading of the script.
[ref.](https://qiita.com/phanect/items/82c85ea4b8f9c373d684)

```suggestion
    <link rel="stylesheet" href="style.css" type="text/css" />
    <script src="./main.js" defer></script> // here!
    <title>TerraceTechフロントエンドエンジニア養成所</title>
```

nits
Incidentally, the `type="text/css"` attribute for CSS loading can be omitted.
[ref.](https://www.tagindex.com/html/page/link_style.html)

p.s.
sorry,`type="text/javascript"` is too !

Reply:
Thank you for your comment Haru-san,

As per your suggestion, I relocated the link of JS within the HTML file.
Please see the latest commit 
96057c52090197748e4cabff8d120d3c726ddf63